util/dvsim/SlurmLauncher.py

```diff
@@ -0,0 +1,146 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import shlex
+import subprocess
+
+from Launcher import ErrorMessage, Launcher, LauncherError
+
+
+SLURM_QUEUE = os.environ.get("SLURM_QUEUE", "hw-m")
+SLURM_MEM = os.environ.get("SLURM_MEM", "16G")
+SLURM_MINCPUS = os.environ.get("SLURM_MINCPUS", "8")
+SLURM_TIMEOUT = os.environ.get("SLURM_TIMEOUT", "240")
+SLURM_CPUS_PER_TASK = os.environ.get("SLURM_CPUS_PER_TASK", "8")
+SLURM_SETUP_CMD = os.environ.get("SLURM_SLURM_SETUP_CMD", "module purge; module load opentitan")
```

Comment:
This particular line seems to be pretty magic and presumably shows us something about the Rivos setup :-) Is there a way that the "setup command" can appear in a log file that isn't checked in?

Reply:
Most probably. Though it is using standard environment modules. What about leaving that as an empty string as default that a user can overwrite?